app/controllers/questionnaire_controller.rb

```diff
@@ -211,6 +213,20 @@ def check_content_hash
     end
   end
 
+  def check_content_empty
+    quest_content = questionnaire_content
```

Comment:
actually quite misleading that we have 'questionnaire_content' and 'set_questionnaire_content' which do quite different things.

Reply:
I think it's because this controller does both response stuff and questionnaire stuff. It should really be two separate controllers. This is on my list of things to fix, but refactoring is a nightmare and there's other stuff that takes priority right now.